components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/dto/KeyManagerPermissionConfigurationDTO.java

```diff
@@ -0,0 +1,43 @@
+package org.wso2.carbon.apimgt.api.dto;
```

Comment:
Let's add the License header here.

Reply:
Fixed in 094e53270f25762671efb82901d371e8a56b8ff0